frontend/src/metabase-lib/v2/limit.ts

```diff
@@ -0,0 +1,15 @@
+import * as MetabaseLibLimit from "cljs/metabase.lib.limit";
```

Comment:
I imagined we'd only import from `metabase.lib.core` as `MetabaseLib`

Reply:
Fixed to use `metabase.lib.core` 👍